.github/workflows/release.yml

```diff
@@ -1,15 +1,12 @@
 name: Release
 
 on:
-  workflow_run:
-    workflows:
-      - "Rust"
-    types:
-      - completed
```

Comment:
This `workflow_run` event also does not have a `github.ref` object for knowing the tag etc either, so it makes it overly complicated trying to jank this together in this way. Instead we'll use the base tag primitive since its simpler.

Reply:
main builds can fail, and you may not have yet noticed when you tag a commit for release.

I think it's still a good idea to wait until the main workflows have succeeded before running the release workflow

Reply:
Yeah that makes sense, I was thinking about this initially but reverted the changes here since it leads into very janky looking CI

Perhaps it's fine though and we'll keep this in

Reply:
Actually nevermind, I'm going to simplify this even more for now and move it into the main file.

I don't think the split is worthwhile for now for the extra complexity when all we want to do is push a container